examples/acm-terraform-blog-part1/terraform/gke.tf

```diff
@@ -0,0 +1,32 @@
+module "enabled_google_apis" {
+  source  = "terraform-google-modules/project-factory/google//modules/project_services"
+  version = "~> 10.0"
+
+  project_id                  = var.project
+  disable_services_on_destroy = false
+
+  activate_apis = [
+    "compute.googleapis.com",
+    "container.googleapis.com",
+    "gkehub.googleapis.com",
+    "anthosconfigmanagement.googleapis.com"
+  ]
+}
+
+resource "random_id" "rand" {
+  byte_length = 4
+}
+
+module "gke" {
+  source                = "terraform-google-modules/kubernetes-engine/google"
+  project_id            = var.project
+  name                  = "sfl-acm-${random_id.rand.hex}"
+  region                = var.region
+  zones                 = [var.zone]
+  initial_node_count    = 4
+  network               = "default"
+  subnetwork            = "default"
```

Comment:
Some orgs may have `constraints/compute.skipDefaultNetworkCreation` org policy which may cause this to fail. We could deploy a vpc or ask users to confirm exists or run `gcloud compute networks create default --subnet-mode=auto`.

Reply:
Thanks! I added a note about this in the readme.